app/_components/trade-routes/single/response/desktop/BodyDesktop.tsx

```diff
@@ -0,0 +1,350 @@
+                  color={selectTextColorAccent}
+                >
+                  Demand:{' '}
+                </Text>
+                <Text color={selectTextColorAccent}>{`${formatThousands(
+                  data.demand,
+                )} units`}</Text>
+              </HStack>
+              <HStack>
+                <Text
+                  className={orbitron.className}
+                  fontWeight={700}
+                  letterSpacing={0.5}
+                  color={selectTextColorAccent}
+                >
+                  Sell Price:{' '}
+                </Text>
+                <Text color={selectTextColorAccent}>{`${formatThousands(
+                  data.sellPrice,
+                )} cr`}</Text>
+              </HStack>
+              <HStack>
+                <Text
+                  className={orbitron.className}
+                  fontWeight={700}
+                  letterSpacing={0.5}
+                  color={selectTextColorAccent}
+                >
+                  Last Updated:
+                </Text>
+                <Text color={selectTextColorAccent}>
+                  {calculateTimeDifference(
+                    data.sellToStationDto.marketUpdateAt,
+                  )}
+                </Text>
+              </HStack>
+            </VStack>
+          </Grid>
+        </CardBody>
+      </Card>
+    </Box>
+  );
+
+  return (
+    <Flex
+      width="100%"
+      fontSize="sm"
+      flexFlow="column"
+      onClick={() => setShowItemCard(!showItemCard)}
+      _even={{
+        background: `${GetColor('grid-accent')}`,
+      }}
+      borderRadius="md"
+      borderColor={GetColor('border')}
+      border={showItemCard ? `1px solid` : 'none'}
+      overflowX="auto"
+      boxShadow="md"
+    >
+      <HStack>
+        <Grid
+          gridTemplateColumns={'2fr 4fr 4fr 1fr'}
+          rowGap="1"
+          padding={4}
+          width="100%"
+        >
+          <VStack alignItems="start" as={GridItem}>
+            <Text fontStyle="italic" fontWeight="700">
+              {`${Math.round(+formatThousands(data.routeDistance))} ly`}
+            </Text>
+            <HStack>
+              <Text className={orbitron.className}>Profit: </Text>
+              <Text color={selectTextColorAccent} fontWeight="700">
+                {` ${formatThousands(data.profit * cargoCapacity)} cr`}
+              </Text>
+            </HStack>
+          </VStack>
+          <VStack alignItems="start" as={GridItem}>
+            <Text className={orbitron.className} fontWeight={700}>
+              From:{' '}
+            </Text>
+            <HStack>
+              <Link
+                as={NextLink}
+                color={selectTextColorStrong}
+                fontWeight={700}
+                marginEnd={2}
+                href="#"
+                display="flex"
+                gap={2}
+              >
+                {truncateString(data.buyFromStationDto.system.name, 15)}
```

Comment:
I've added a utility function to truncate large strings of system and station names.
Mostly to clean up the UI on mobile devices.

When the user expands a grid item to show the card with more details, these names are repeated on mobile devices as larger strings.

refer to the images in this PR for examples